### ISSUE TITLE
Vue 3

### ISSUE THREAD

**Issue by szpetny** [widgetti/ipyvuetify#232]
Hi, are you planning to upgrade to vue 3?

**Comment by mariobuikhuizen** [comment]
Yes, we want to upgrade to Vue 3 at some point, but we have no timeline for this yet.

Could you tell us something about why you want/need this?

**Comment by szpetny** [comment]
Thank you for a prompt answer. 
We are starting a new project, which should end up as a pleasant UI. 
And so we found your lib and would like to use it, because it is pretty neat and would let us incorporate nice modern look and some features of vue (thanks for nice work btw :)). 
I just wonder if it is maintainable in the future, in case an upgrade to a newer version will be very needed.
Hence the question :)
